### PR TITLE
add healthlocker to the apps directory

### PR DESCRIPTION
**Pull request by RobStallion**
adds the `healthlocker app` to the list of apps in the umbrella project #2

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -21,6 +21,6 @@ defmodule HlUmbrella.Mixfile do
   # Dependencies listed here are available only for this project
   # and cannot be accessed from applications inside the apps folder
   defp deps do
-    []
+    [{:exrm, ">= 0.16.0", warn_missing: false}]
```
Comment (nelsonic) [inline mix.exs]: @RobStallion [`:exrm |> :distillery`](https://github.com/bitwalker/exrm/tree/21bd28da70f818d895aae76e40966e91580d4667#elixir-release-manager) 😉

**Comment by nelsonic** [comment]
@RobStallion this outline looks good please remove `wip` label if ready for merging thanks. 👍

File: mix.exs (hunk 1 above)
Comment (RobStallion) [reply mix.exs]: Thanks for spotting that 👍

**Review by nelsonic — APPROVED** [review]
@RobStallion looks good! thanks! 🎉

**Pull request merged**